packages/camera/camera_android/android/src/main/java/io/flutter/plugins/camera/CameraApiImpl.java

```diff
@@ -0,0 +1,354 @@
+      camera.setExposurePoint(result, point == null ? null : new Point(point.getX(), point.getY()));
+    } catch (Exception e) {
+      handleException(e, result);
+    }
+  }
+
+  @NonNull
+  @Override
+  public Double getMinExposureOffset() {
+    return camera.getMinExposureOffset();
+  }
+
+  @NonNull
+  @Override
+  public Double getMaxExposureOffset() {
+    return camera.getMaxExposureOffset();
+  }
+
+  @NonNull
+  @Override
+  public Double getExposureOffsetStepSize() {
+    return camera.getExposureOffsetStepSize();
+  }
+
+  @Override
+  public void setExposureOffset(@NonNull Double offset, @NonNull Messages.Result<Double> result) {
+    try {
+      camera.setExposureOffset(result, offset);
+    } catch (Exception e) {
+      handleException(e, result);
+    }
+  }
+
+  @Override
+  public void setFocusMode(@NonNull Messages.PlatformFocusMode focusMode) {
+    camera.setFocusMode(CameraUtils.focusModeFromPigeon(focusMode));
+  }
+
+  @Override
+  public void setFocusPoint(
+      @Nullable Messages.PlatformPoint point, @NonNull Messages.VoidResult result) {
+    try {
+      camera.setFocusPoint(result, point == null ? null : new Point(point.getX(), point.getY()));
+    } catch (Exception e) {
+      handleException(e, result);
+    }
+  }
+
+  @Override
+  public void startImageStream() {
+    try {
+      camera.startPreviewWithImageStream(imageStreamChannel);
+    } catch (CameraAccessException e) {
+      throw new Messages.FlutterError("CameraAccessException", e.getMessage(), null);
+    }
+  }
+
+  @Override
+  public void stopImageStream() {
+    try {
+      camera.startPreview();
+    } catch (Exception e) {
+      throw new Messages.FlutterError(e.getClass().getName(), e.getMessage(), null);
+    }
+  }
+
+  @NonNull
+  @Override
+  public Double getMaxZoomLevel() {
+    assert camera != null;
+    return (double) camera.getMaxZoomLevel();
+  }
+
+  @NonNull
+  @Override
+  public Double getMinZoomLevel() {
+    assert camera != null;
+    return (double) camera.getMinZoomLevel();
+  }
+
+  @Override
+  public void setZoomLevel(@NonNull Double zoom, @NonNull Messages.VoidResult result) {
+    assert camera != null;
+    camera.setZoomLevel(result, zoom.floatValue());
+  }
+
+  @Override
+  public void lockCaptureOrientation(
+      @NonNull Messages.PlatformDeviceOrientation platformOrientation) {
+    camera.lockCaptureOrientation(CameraUtils.orientationFromPigeon(platformOrientation));
+  }
+
+  @Override
+  public void unlockCaptureOrientation() {
+    camera.unlockCaptureOrientation();
+  }
+
+  @Override
+  public void pausePreview() {
+    try {
+      camera.pausePreview();
+    } catch (CameraAccessException e) {
+      throw new Messages.FlutterError("CameraAccessException", e.getMessage(), null);
+    }
+  }
+
+  @Override
+  public void resumePreview() {
+    try {
+      camera.resumePreview();
+    } catch (Exception e) {
+      throw new Messages.FlutterError("CameraAccessException", e.getMessage(), null);
```

Comment:
I missed this, but here and below you are catching `Exception` but reporting it as a `CameraAccessException`. I don't think you need these two handlers at all since you aren't doing anything specific with the exceptions, and these are sync methods, so the generic Pigeon-level handler should be fine.

Reply:
Same with `stopImageStream`.

Reply:
`stopImageStream` calls `startPreview`, which can throw two types of non-runtime Exception, so it needs to be wrapped in a try/catch to compile. I'm not sure what you're referring to by "and below", but I made the change to this method and incorporated the feedback from your other comment.